### PR TITLE
Add inventory reference to metrics ansible playbook calls

### PR DESCRIPTION
**Pull request by ghost**
This is a follow up to PR https://github.com/openshift/openshift-docs/pull/6005.

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@lpsantil Please review. Also, what doc versions does this change apply to?

@openshift/team-documentation Please peer review.

**Comment by lpsantil** [comment]
@tmorriso-rh This change applies to 3.5+

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -68,7 +69,8 @@ endif::[]
 
 An Ansible playbook is available to deploy and upgrade cluster metrics. You
 should familiarize yourself with the
-xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Advanced Installation] section. This provides information for prepairing to use Ansible
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Advanced
+Installation] section. This provides information for prepairing to use Ansible
```
Comment (mburke5678) [inline install_config/cluster_metrics.adoc]: s/prepairing/preparing

**Comment by mburke5678** [comment]
@tmorriso-rh One nit. LGTM

**Comment by tmorriso-rh** [comment]
No rev history needed.

**Pull request merged**